### PR TITLE
Added Troubleshoot documents, return management

### PR DESCRIPTION
**Pull request by Amir-HC**
Updated TOC to include shipping method mapping page
Updated product page in troubleshooting top add code blocks for buttons and pages

### CODE REVIEW

**Review by dt2patel — APPROVED** [review]

File: troubleshooting/Shopify/inventory.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#  Inventory Auditing
+
+**Scenario:** Inventory Not Synced from HotWax Commerce to Shopify
+
+Sometimes retailers may encounter inventory disparities between HotWax Commerce and Shopify Inventory. This discrepancy could result in inventory appearing out of stock in Shopify despite being available in HotWax Commerce.
```
Comment (Dhiraj1405) [inline troubleshooting/Shopify/inventory.md]: Write the other half as an example.

File: troubleshooting/Shopify/inventory.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Steps to Troubleshoot
+
+### Verify in HotWax Commerce
+
+1. **Check Job Status:**
+
+   - Navigate to the `HotWax Commerce Job Manager App`.
+   - Locate the job named `Upload recent inventory change`. This job is responsible for updating all inventory changes in Shopify that have occurred in HotWax Commerce since the last time the job ran.
```
Comment (Dhiraj1405) [inline troubleshooting/Shopify/inventory.md]: As both of these jobs are responsible for inventory changes, we should write in that way and ask user to verify on basis of its use case.

File: troubleshooting/Shopify/inventory.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+   - Ensure that the job is running. If not scheduled, schedule the job according to your preferred frequency.
+   - You can also execute the `Hard Sync` job in HotWax Commerce. This job updates inventory for all the products irrespective of the inventory changes, eliminating any discrepancy between HotWax Commerce and Shopify.
+
+2. **Inspect Inventory Configurations:**
+
+Retailers can set rules such as safety stocks, thresholds, and exclude facilities to prevent overselling on Shopify. Follow these steps to verify the online ATP of the inventory that is being synchronized between HotWax Commerce and Shopify:
+
+   - In HotWax Commerce, Go to `Products` > `Find Products` > `Product Detail page` > `Product Inventory View` page.
+   - Check inventory configurations, including safety stocks, thresholds, reserved inventory, and excluded facilities.
+   - Verify the `online ATP` of products on the inventory dashboard, if the `online ATP` is 0 then the product will appear out of stock in Shopify.
```
Comment (Dhiraj1405) [inline troubleshooting/Shopify/inventory.md]: We should make it generic, this is reflecting a specific use case

File: troubleshooting/Shopify/inventory.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+1. **Check Online ATP in Inventory Dashboard:**
+   - Navigate to Shopify Admin Panel.
+   - Go to `Products` > `Product detail` page.
+   - Click on the variant of the product that requires inventory verification.
+
+2. **Verify Inventory Adjustment History:**
+   - Within the variant's `inventory` section, click on the `adjustment history` button.
+   - Examine historical inventory adjustments. Ensure they reflect recent changes.
+   - If historical changes show old records or there are no inventory records, go to the `HotWax Commerce Job manager App` and run the `Hard Sync` job.
```
Comment (Dhiraj1405) [inline troubleshooting/Shopify/inventory.md]: Write both jobs specific to use case

**Pull request merged**